src/content/general/management-clusters/index.md

```diff
@@ -25,21 +25,36 @@ As well as operators, our management clusters run supporting components such as
 
 Our workload clusters are versioned using [workload cluster releases]({{< relref "/general/releases" >}}). From a workload cluster point of view, upgrades are described in [cluster upgrades]({{< relref "/general/cluster-upgrades" >}}).
 
-When we publish a new Giant Swarm release, we create a new release custom resource and any new operator versions are deployed to the management cluster.
-We deploy new instances of the operators with the new version to avoid impacting existing workload clusters. These new operators will not become active until existing workload clusters are upgraded or a new cluster is created.
+When we publish a new Giant Swarm release, we create a new [Release Custom Resource]({{< relref "/ui-api/management-api/crd/releases.release.giantswarm.io/" >}}) and any new operator versions are deployed to the management cluster.
```

Comment:
```suggestion
When we publish a new workload cluster release, we create a new [Release Custom Resource]({{< relref "/ui-api/management-api/crd/releases.release.giantswarm.io/" >}}) and any new operator versions are deployed to the management cluster.
```

Reply:
i don't know how, but this got resolved somewhere :D